cardano-db-sync/src/Cardano/DbSync/Era/Universal/Insert/Tx.hs

```diff
@@ -77,6 +77,7 @@ insertTx syncEnv isMember blkId epochNo slotNo applyResult blockIndex tx grouped
   let !mdeposits = if not (Generic.txValidContract tx) then Just (Coin 0) else lookupDepositsMap txHash (apDepositsMap applyResult)
   let !outSum = fromIntegral $ unCoin $ Generic.txOutSum tx
       !withdrawalSum = fromIntegral $ unCoin $ Generic.txWithdrawalSum tx
+      !treasuryDonation = fromIntegral . unCoin $ Generic.txTreasuryDonation tx
```

Comment:
IIrc we wanted to use this in the deposit calculation for a specifc case.

Reply:
Thanks for the reminder, I just updated it to be subtracted from the inputs. However, I couldn't find a way to test it, so please carefully re-review